### DISCUSSION TITLE
How to remove theme properties?

### DISCUSSION THREAD

**Discussion by rahrang** [chakra-ui/chakra-ui#4275] [Q&A] [answered]
Hi there,

I have a custom theme I'm using with Chakra's `extendTheme` function. Chakra's default theme has a few properties I want to remove so that I don't use them. For instance, I'd like to remove the brand colors (`facebook`, `instagram`, `whatsapp`, etc.), the `space` keys from 9 and up.

What I'm doing is

```ts
import { extendTheme } from '@chakra-ui/react'

const colors = {
  facebook: null // I've also tried {} and undefined
}

const overrides = {
  colors,
}

const theme = extendTheme(overrides)
```

None of these work. `null` causes a TypeScript issues when passing in `overrides` to `extendTheme`. `{}` and `undefined` get overwritten by the properties in Chakra's default theme.

**Comment by nikolovlazar** [comment, marked as answer]
You could `delete` them after the `extendTheme` sentence:
```typescript
const theme = extendTheme({
   ...
});

delete theme.colors.facebook;
delete theme.colors.linkedin;
delete theme.colors.messenger;
delete theme.colors.telegram;
delete theme.colors.whatsapp;
delete theme.colors.twitter;

export default theme;
```

I've also created a [codesandbox](https://codesandbox.io/s/delete-unused-styles-y1ekp) that you can play with. Note the commented out `whatsapp` and `twitter` colors, they're the only ones present in the theme. Check out the Console output to see the whole `theme` object after extending it.

**Reply by rahrang** [reply]
this works great, thanks @lazarnikolov94!

**Reply by nikolovlazar** [reply]
It seems that I've forgotten to paste the codesandbox link: https://codesandbox.io/s/delete-unused-styles-y1ekp Sorry, my bad 😁
